### DISCUSSION TITLE
The lazer tournament client should use the default mod icons in the tourney client "Round Editor" screen if no icons are present in the tournament folder

### DISCUSSION THREAD

**Discussion by 0x84f** [ppy/osu#17538] [Q&A] [answered]
Currently if you do not have a separate icon pack in the tournament folder there will be no mod icons at all when adding mods to beatmap pools in the "Rounds Editor", example can be seen [here](https://i.imgur.com/BTleIBw.jpg).
When I first installed the tourney client it was using the default icons, but once I rebooted it, it stopped showing them from that point forwards. Also previously the default icons would be added if you added the mods first then the beatmap ID, but that also doesn't work anymore.

**Comment by peppy** [comment, marked as answer]
The code looks correct to provide defaults as fallback. Could you please share your bracket and tournament resources zipped up?

**Reply by 0x84f** [reply]
[Here you go!](https://2127.s-ul.eu/suNqudrO.zip)

**Reply by peppy** [reply]
Thanks, I was able to reproduce. This will be fixed via #17574.
